app/models/case_contact.rb

```diff
@@ -39,19 +45,6 @@ class CaseContact < ApplicationRecord
 
   after_save_commit ::CaseContactMetadataCallback.new
 
-  # Corresponds to the steps in the controller, so validations for certain columns can happen at those steps.
-  # These steps must be listed in order and have an html template in case_contacts/form.
-  FORM_STEPS = %i[details notes expenses].freeze
-  enum status: (%w[started active] + FORM_STEPS.map(&:to_s)).zip((%w[started active] + FORM_STEPS.map(&:to_s))).to_h
```

Comment:
This enum is written fancily (& confusingly) - evaluates to:
```ruby
{
  "started"=>"started", 
  "active"=>"active", 
  "details"=>"details", 
  "notes"=>"notes", 
  "expenses"=>"expenses"
}
```
but it is also is on a **string column**, which is very non-standard way for enums, normally right hand side/column would be integer. [See Rails Enum docs](https://api.rubyonrails.org/v7.1.3.4/classes/ActiveRecord/Enum.html). 

It defines `active?` and similar for each member. We can get rid of it and define methods explicitly (addition below).